src/Arcus.Scripting.Tests.Unit/Arcus.Scripting.ActiveDirectory.ps1

```diff
@@ -0,0 +1,751 @@
+﻿Import-Module Microsoft.Graph.Applications
```

Comment:
Great job in providing some failure/success examples and to organize them in contexts. Really helps maintainability.